.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     'no-unused-expressions': 0,
     'chai-friendly/no-unused-expressions': 2,
     'prettier/prettier': 'error',
+    'class-methods-use-this': ['error', { exceptMethods: ["isVideoUrl"] }],
```

Comment:
Instead of doing this in the global config, I think it would be better to disable it in-line in src/base.js:

Adding a comment like this to the end of the line that is failing should do it

```
// eslint-disable-line class-methods-use-this
```

Reply:
Encountered this error when I added it to the end of the line, so I added the exclusion before with `// eslint-disable-next-line`

![exclusion-error](https://user-images.githubusercontent.com/26092352/105339142-34411c80-5b91-11eb-83db-915c43b17aaa.png)

Reply:
perfect!